src/MobileApps/MyDriving/MyDriving.Shared/PlatformLogger.cs

```diff
@@ -22,12 +22,6 @@ public override void Identify(string uid, IDictionary<string, string> table = nu
             base.Identify(uid, table);
         }
 
-        public override void WriteLine(string line)
```

Comment:
why delete it?
We can keep both. One for server logging, one for local debugging, right?